translation/templates.rst

```diff
@@ -39,6 +39,11 @@ works when you use a placeholder following the ``%var%`` pattern.
 
     If you need to use the percent character (``%``) in a string, escape it by
     doubling it: ``{% trans %}Percent: %percent%%%{% endtrans %}``
+    
+.. deprecated:: 4.2
```

Comment:
Is there another possible type for this? I think that a BC break is not a deprecation.

Reply:
Yes there is versionchanged directive 

cc @javiereguiluz

Reply:
now using the versionchanged directive